src/AzureExtension/DataManager/AzureDataManager.cs

```diff
@@ -102,7 +102,7 @@ public AzureDataManager(string identifier, DataStoreOptions dataStoreOptions)
             Log.Logger()?.ReportWarn(Name, InstanceName, "Failed setting DeveloperId change handler.", ex);
         }
 
-        if (Instances.ContainsKey(InstanceName))
+        if (Instances.TryGetValue(InstanceName, out var value))

```

Comment:
Nit: I think you can replace `Instances[InstanceName]` on line 111 with `value`, yeah?

Reply:
You are correct.  I totally missed this.